runtime/ftplugin/zathurarc.vim

```diff
@@ -0,0 +1,21 @@
+" Vim filetype plugin file
+" Language:             Zathurarc
+" Maintainer:           Wu, Zhenyu <wuzhenyu@ustc.edu>
+" URL:                  https://pwmt.org/projects/zathura/documentation/
+" Latest Revision:      2024-04-02
+
+if exists('b:did_ftplugin')
+  finish
+endif
+let b:did_ftplugin = 1
+
+let s:save_cpoptions = &cpoptions
+set cpoptions&vim
+
```

Comment:
How about adding a [&keywordprg](https://github.com/Konfekt/vim-keyword-lookup/blob/46edeedadac4d30f8764dce1a7d53ceda66cec82/ftdetect/keywordprg.vim#L18)

Reply:
I suppose you meant line 17 instead?

Reply:
Shouldn't the "URL" field point at some kind of upstream repo, rather than the zathura docs?

Reply:
> some kind of upstream repo

Should be `Upstream:`?

Reply:
URL should point to your upstream repo if you have one. You can rename the current URL to Documentation I guess.